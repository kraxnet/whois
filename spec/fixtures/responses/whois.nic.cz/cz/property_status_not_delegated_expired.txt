%  (c) 2006-2011 CZ.NIC, z.s.p.o.
%
% Intended use of supplied data and information
%
% Data contained in the domain name register, as well as information
% supplied through public information services of CZ.NIC association,
% are appointed only for purposes connected with Internet network
% administration and operation, or for the purpose of legal or other
% similar proceedings, in process as regards a matter connected
% particularly with holding and using a concrete domain name.
%
% Full text available at:
% http://www.nic.cz/page/306/intended-use-of-supplied-data-and-information/
%
% See also a search service at http://www.nic.cz/whois/
% 
% 
% Whoisd Server Version: 3.4.0
% Timestamp: Sat May 26 04:09:24 2012

domain:       306.cz
registrant:   DCM
nsset:        KRAXNET
registrar:    REG-1X
status:       Domain is not generated into zone
status:       Expired
registered:   27.03.2009 14:25:19
changed:      22.11.2009 22:38:56
expire:       27.03.2012

contact:      DCM
name:         Martin Ladyr
address:      Kladenska 94/21
address:      Praha 6
address:      16000
address:      CZ
e-mail:       domenovecentrum@gmail.com
registrar:    REG-ZONER
created:      07.01.2009 03:10:22

nsset:        KRAXNET
nserver:      ns.kraxnet.com
nserver:      ns.kraxnet.cz (178.217.247.2, 2a02:1360::56)
tech-c:       SB:KRAXNET
registrar:    REG-KRAXNET
created:      03.11.2009 17:58:56
changed:      19.09.2011 17:48:01

contact:      SB:KRAXNET
org:          KRAXNET s.r.o.
name:         Jiří Kubíček
address:      Kamenická 26
address:      Praha 7
address:      170 00
address:      CZ
e-mail:       info@kraxnet.cz
registrar:    REG-KRAXNET
created:      10.08.2001 22:13:00
changed:      12.09.2003 15:51:00
