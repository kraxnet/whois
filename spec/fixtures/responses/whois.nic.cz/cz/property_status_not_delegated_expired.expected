#status
  %s == [:out_of_zone, :expired]

#available?
  %s == false

#registered?
  %s == true
